proof_of_concept/rest/internal_client.py

```diff
@@ -78,11 +83,30 @@ def submit_job(self, job: Job) -> str:
             raise RuntimeError(f'Error submitting job: {r.text}')
         if 'location' not in r.headers:
             raise RuntimeError('Invalid server response when submitting job')
+
+        # Protect against malicious servers redirecting us elsewhere
+        std_ports = {'http': 80, 'https': 443}
```

Comment:
Factor out a constant maybe?

Reply:
Haha :-). How about this:
```
HTTP_SCHEMA = 'http'
HTTP_PORT = 80
HTTPS_SCHEMA = 'https'
HTTPS_PORT = 443
STD_PORTS = {HTTP_SCHEMA: HTTP_PORT, HTTPS_SCHEMA: HTTPS_PORT}
```

Reply:
That is maybe too much?

For me
```python
STANDARD_PORTS = {'http': 80, 'https': 443}
```
would be good enough.

Would the overall software quality of the project really be affected by what you choose here? Probably not... 🦊

Reply:
NEEEEDS_MOOOOOOOR_CONSTANTS!

You know, I could make all the strings in my error messages constants...Hmmmm.